templates/default/sql/dml/1.3.0.sql.erb

```diff
@@ -11,3 +11,10 @@ DELETE FROM `hopsworks`.`variables` WHERE `id`='hiveserver_ext_hostname';
 DELETE FROM `hopsworks`.`variables` WHERE `id`='hiveserver_ssl_hostname';
 REPLACE INTO `hopsworks`.`variables` VALUES ("hopsworks_enterprise", "<%= node['install']['enterprise']['install']%>");
 REPLACE INTO `hopsworks`.`variables` VALUES ("kubernetes_installed", "<%= node['install']['kubernetes'] %>");
+INSERT INTO `hopsworks`.`variables` VALUES ("yarn_runtime", "docker");
+INSERT INTO `hopsworks`.`variables` VALUES ("docker_mounts", "<%= node['hops']['docker']['mounts'] %>");
```

Comment:
Please use `REPLACE INTO` as these files are run at every upgrade. So the next upgrade there will be a PK violation if you just use `INSERT INTO`

Reply:
fixed